hedyorg/hedy
# [CHORE] Move docs to /docs directory

**Description**

Moves documentation files to `/docs` and updates references in markdown files.

**Part of #4447 **

**How to test**

* Go to https://github.com/jtwaleson/hedy
* See that license file is correctly shown on the right
* See that citation file is correctly shown
* When opening an new issue, the issue templates work
* Following the links in the readme to DESIGN.md and CONTRIBUTING.md work

**Checklist**
